SQL/paradise_schema.sql

```diff
@@ -250,6 +250,7 @@ CREATE TABLE `player` (
   `nanoui_fancy` smallint(4) DEFAULT '1',
   `show_ghostitem_attack` smallint(4) DEFAULT '1',
   `lastchangelog` varchar(32) NOT NULL DEFAULT '0',
+  `exp` mediumtext,
```

Comment:
What will this be? What will be inserted into this row, and why is it ´mediumtext´